livestream-compose-sample/src/main/java/io/getstream/livestream/compose/ui/LiveStreamComment.kt

```diff
@@ -0,0 +1,78 @@
+package io.getstream.livestream.compose.ui
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import io.getstream.chat.android.client.models.name
+import io.getstream.chat.android.compose.state.messages.items.MessageItem
+import io.getstream.chat.android.compose.ui.common.MessageBubble
+import io.getstream.chat.android.compose.ui.common.avatar.UserAvatar
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.livestream.compose.getTimeAgo
+import java.text.SimpleDateFormat
+import java.util.Date
+
+/**
+ * View component to add custom live stream channels screen.
```

Comment:
This comment is C/P and doesn't make sense. How is this component adding live stream channels screen? This is a component that's used to customize the messages on that screen. We need to be specific with our docs.

Reply:
Updated